api/rs/slint/mcu.md

```diff
@@ -225,24 +221,22 @@ loop {
 
 ### The Renderer
 
-In desktop and embedded environments, Slint typically uses operating system provided, often hardware-accelerated APIs to render the user interface.
+In desktop and embedded environments, Slint typically uses operating system provided APIs to render the user interface. These APIs are often hardware accelerated.
 In contrast, most MCUs don't have dedicated chips to render advanced graphics. Instead, the CPU is responsible for computing the colors of each
 pixels on the screen. This is called software rendering, and Slint provides a software renderer for this task.
```

Comment:
I'm not sure it is so good to say that "most MCUs don't have dedicated chips to render advanced graphics", while many do. (eg, DMA2D or other DMA-like accelerator)
This could confuse people who would think they cannot use the software renderer to use DMA2D, for example

My suggestion:

```suggestion
In desktop and embedded environments, Slint typically uses operating system provided APIs to render the user interface using the GPU.
In contrast, most MCUs don't have GPUs. Instead, the all the rendering is done by sotfware on the CPU. This is called software rendering, and Slint provides a SoftwareRenderer for this task.
```